build.sbt

```diff
@@ -124,6 +128,30 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     name := "otel4s-core"
   )
 
+lazy val `sdk-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .enablePlugins(BuildInfoPlugin)
+  .in(file("sdk/common"))
+  .dependsOn(`core-common`, semconv)
+  .settings(
+    name := "otel4s-sdk-common",
+    startYear := Some(2023),
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect-kernel" % CatsEffectVersion,
+      "org.typelevel" %%% "cats-mtl" % CatsMtlVersion,
+      "org.typelevel" %%% "cats-laws" % CatsVersion % Test,
+      "org.typelevel" %%% "discipline-munit" % DisciplineMUnitVersion % Test,
+      "org.scalameta" %%% "munit" % MUnitVersion % Test,
+      "org.scalameta" %%% "munit-scalacheck" % MUnitVersion % Test,
+    ),
+    buildInfoPackage := "org.typelevel.otel4s.sdk",
+    buildInfoOptions += sbtbuildinfo.BuildInfoOption.PackagePrivate,
+    buildInfoKeys := Seq[BuildInfoKey](
+      "openTelemetrySdkVersion" -> OpenTelemetryVersion
```

Comment:
Isn't this the version of Java OTel, why are we including this as a build-info key?

Reply:
I've replaced with the SDK version from build.sbt